torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -277,28 +277,68 @@ Operation BroadOp(const Node* node) {
     auto a_size = a.sizes();
     if (a_size.equals(b_size)) {
```

Comment:
Is it possible we still want to reorder one of the tensors if one is in a blocked format and one is in a public format even if they are the same size ?

Reply:
yes, but it seems to fall outside the scope of this PR, since it could happen even without inserting explicit broadcasts say if we just use python API with MKLDNN tensors. I think the general case should be handled inside of binary ops (for eager).